### PR TITLE
Feature/136 prepare for integration

### LINKED ISSUE #136: Prepare for integration with new login

### PR DESCRIPTION
**Pull request by dami-gg**
Closes #136

### CODE REVIEW

File: src/root.module.js [hunk 1]
```diff
@@ -1,24 +1,21 @@
...
-]);
-
-RootModule.config(($ngReduxProvider) => {
-  let middleware = [thunk];
-
-  $ngReduxProvider.createStoreWith(reducers, middleware);
-});
+])
+  .config(RootConfig)
+  .constant('SCN_REDUCERS', reducersObject);
```
Comment (frontendherodk) [inline src/root.module.js]: Where is this one needed?

Comment (dami-gg) [inline src/root.module.js]: in any client app which uses redux. It's the only way we can avoid any collisions.

Comment (dami-gg) [inline src/root.module.js]: check the related PR on the user creation MVP

**Comment by frontendherodk** [comment]
👍

**Comment by dami-gg** [comment]
:+1:

**Pull request merged**